wagtail/models/i18n.py

```diff
@@ -174,13 +174,143 @@ def is_active(self) -> bool:
             return self.is_default
 
 
+class TranslatableQuerySetMixin:
+    """
+    QuerySet mixin for translatable models.
+
+    This mixin provides methods for query sets of translatable models. If your
+    translatable model inherits from
+    :py:class:`TranslatableMixin <wagtail.models.TranslatableMixin>` and defines a
+    custom manager that inherits from ``models.QuerySet``, be sure to also inherit
+    from this queryset mixin to retain the features of this mixin.
+
+    .. code-block:: python
+
+        class MyTranslatableModelQuerySet(TranslatableQuerySetMixin, models.QuerySet):
+            pass
+
+        class MyTranslatableModel(TranslatableMixin):
+            objects = MyTranslatableModelQuerySet.as_manager()
+
+    If your translatable model does not define a custom manager ``objects``, you won't
+    need this mixin. The ``TranslatableMixin`` already provides a manager that inherits
+    from this mixin.
+    """
+
+    def localized(
+        self: models.QuerySet,
+        keep_order: bool = False,
```

Comment:
```suggestion
        preserve_order: bool = False,
```

Reply:
That does sound better 😅

Reply:
509afb27612828e7c8b4dc92d1b0e8dcd133ec09